setup.py

```diff
@@ -96,7 +96,7 @@ def run(self):
 gs = ["google-cloud-storage==1.19.0"]
 gdrive = ["pydrive2>=1.6.3", "six >= 1.13.0"]
 s3 = ["boto3>=1.9.201"]
-azure = ["azure-storage-blob>=12.0", "knack"]
+azure = ["adlfs>=0.6.2", "knack"]
```

Comment:
a thing to check - it brings more dependencies? I remember we had some troubles with some Azure libs .. @efiop should have a better idea :)

Reply:
We had problems with old azure-storage-blob, now that's adlfs's problem to handle it properly and they do :) More dependencies also because it supports datalake, which we've been asked about before but never added support for(we'll need to do some adjustments to support it with fsspec too). So that's also justifiable.